components/brave_rewards/browser/rewards_service_impl.cc

```diff
@@ -2200,8 +2200,19 @@ bool SavePendingContributionOnFileTaskRunner(PublisherInfoDatabase* backend,
   return backend->InsertPendingContribution(list);
 }
 
+void RewardsServiceImpl::OnCallBackPendingContribution(double amount) {
+  for (auto& observer : observers_)
+    observer.OnPendingContributionsTotal(this, amount);
+}
+
 void RewardsServiceImpl::OnSavePendingContribution(bool result) {
-  // TODO(nejczdovc) add callback with db result
+  if (result) {
+    GetPendingContributionsTotal(base::Bind(
```

Comment:
we shouldn't be making queries to update the UI here. This should trigger a PendingContributionSaved method on the observer and the UI should call GetPendingContributionsTotal in response to that

Reply:
done